docs/block-api.md

```diff
@@ -93,7 +105,7 @@ Sometimes a block could have aliases that help users discover it while searching
 
 ```js
 // Make it easier to discover a block with keyword aliases
-keywords: [ __( 'read' ) ],
+keywords: [ 'image', 'photo', 'pics' ],
```

Comment:
Our examples should be localised, encouraging authors to do the same for their blocks. I'll add it here though 😄

Reply:
Didn't localise following the pattern, also for new users, it might be confusing. For `title` and `description` we are using plain string. For consistency, we should update this across handbook.

Added to my `TODO's` for next handbook reading/updating sprint.